jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -12,7 +12,6 @@ def buildExclusions = [
   // Machine Label,                   Build Type,  Node Version
 
   // Linux -------------------------------------------------
-  [ /^centos5/,                       anyType,     gte(8)  ],
```

Comment:
We need to make sure that all jobs referencing these nodes are updated before committing this otherwise they will start scheduling jobs against them (since this script is "don't allow if" so if the line is removed it is allowed).